_episodes/05-which-cloud.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "Which Cloud for my data?"
+teaching: 10
+exercises: 20
+questions:
+- What cloud resources are available?
+- How do I choose my resources?
+objectives:
+- Describe the advantages and disadvantages of various cloud computing platforms.
+- Determine what resources a project will require
+- Match required resources to computing platforms
+---
+
+
+## Cloud platform choices
+
+There are several cloud providers to choose from. Some scientific clouds may either be free or allocate resources competitively. Commercial clouds can be very powerful, but choice can be overwhelming. We will cover as much as we can, but you will ultimately need to learn things not covered here so see the documentation below:
+
+The major tradeoff between platforms is between flexibility and cost. Generally speaking,
+services that allow you more flexibility and autonomy will be more expensive than
+highly managed services.
+
+### University/Corporate Computing Clusters
+
+Many universities and businesses operate their own computing clusters that are available to students and staff
+at low or no cost. If your employer maintains a computing cluster, this will almost always be the least
+expensive option.
+
+However, most HPCs put limits on:
```

Comment:
I think this is the first time we use the acronym "HPC" so should spell it out here.